Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt

```diff
@@ -50,7 +50,10 @@ import com.example.jetsnack.model.Filter
 import com.example.jetsnack.ui.theme.JetsnackTheme
 
 @Composable
-fun FilterBar(filters: List<Filter>) {
+fun FilterBar(
+    filters: List<Filter>,
+    onFilterClicked: () -> Unit
```

Comment:
Should the lambda accept the selected filter?

Reply:
Also 
```suggestion
    onFilterClick: () -> Unit
```

Reply:
The lambda is not actually receiving a filter, it is opening the more filters modal.  I could rename it to make it clear.

Reply:
Ahhhh, yes perhaps `onShowFilters` or similar?

Reply:
I think the name should be onFilterClicked.